src/component/mxgraph/shape/gateway-shapes.ts

```diff
@@ -73,6 +54,23 @@ abstract class GatewayShape extends mxRhombus {
     const yTranslation = parentY + parentHeight / 4;
     c.translate(xTranslation, yTranslation);
   }
+
+  protected drawCrossSymbol(canvas: MxScaleFactorCanvas, parentWidth: number, parentHeight: number): void {
```

Comment:
Not important:
- sometimes we use _paint_ word, sometimes _draw_ word
- sometimes we use _icon_ word, sometimes _symbol_ word

It would be better to use the same words everywhere ;)

Reply:
I agree with @csouchet 
AFAIK, we agreed to use `icon` for the main BPMN element identifiers and `marker` for additional render info (probably only for `Activities`)
See https://github.com/process-analytics/bpmn-visualization-js/pull/183#discussion_r409383875

Reply:
I agree also
icon is already chosen as @tbouffard pointed out
for the other word I would go for 'paint' as 'draw' is widely used. Let's make 'paint' our hidden signature :)